tasks/Functions-client/buildOffchainSecrets.js

```diff
@@ -8,40 +9,35 @@ task(
   "functions-build-offchain-secrets",
   "Builds an off-chain secrets object that can be uploaded and referenced via URL"
 )
-  .addOptionalParam("output", "Output file name (defaults to offchain-secrets.json)")
-  .addOptionalParam(
-    "configpath",
-    "Path to Functions request config file",
-    `${__dirname}/../../Functions-request-config.js`,
-    types.string
-  )
+  .addOptionalParam("output", "Output JSON file name (defaults to offchain-secrets.json)")
   .setAction(async (taskArgs) => {
     if (network.name === "hardhat") {
       throw Error("This command cannot be used on a local hardhat chain.")
     }
 
-    const requestConfig = require(path.isAbsolute(taskArgs.configpath)
```

Comment:
This `requestConfig` should contain the secrets we want to use.  It shouldn't be removed.